server/src/main/java/org/opensearch/index/OpenSearchTieredMergePolicy.java

```diff
@@ -80,7 +80,7 @@ public void setForceMergeDeletesPctAllowed(double forceMergeDeletesPctAllowed) {
     }
 
     public double getForceMergeDeletesPctAllowed() {
```

Comment:
@msfroh trying to understand the difference here, this is the setter does update both policies

```
    public void setForceMergeDeletesPctAllowed(double forceMergeDeletesPctAllowed) {
        regularMergePolicy.setForceMergeDeletesPctAllowed(forceMergeDeletesPctAllowed);
        forcedMergePolicy.setForceMergeDeletesPctAllowed(forceMergeDeletesPctAllowed);
    }

```

so it shouldn't matter with one to query, right?

Reply:
True -- both MPs will give the same value. Functionally, this line changes nothing.

That said, I figured since I'm changing the logic so `regularMergePolicy` is the one actually finding forced delete merges, it's the MP whose opinion matters (which I think is similar is spirit to the old logic that was reading the value from `forcedMergePolicy` while most getters read from `regularMergePolicy`).

Reply:
> That said, I figured since I'm changing the logic so `regularMergePolicy` is the one actually finding forced delete merges,

Any negative consequences of this behavior change (finding delete merges)?

Reply:
IMO the biggest negative consequence is that the behavior changes. Potentially there were users who got used to `only_expunge_deletes` producing large segments.

I suspect that more users have been surprised by the current behavior, though, and would expect the new behavior.

I feel like if people want the "ignore max segment size" behavior, they'll do a force merge to a specified number of segments (usually 1).